_data/projects/search/releases/7.0/7.0.0.CR1.yml

```
@@ -0,0 +1,10 @@
+date: 2023-09-05
+announcement_url: https://in.relation.to/2023/10/23/hibernate-search-7-0-0-CR1
+summary: >-
+  Compatibility with Hibernate ORM's discriminator-based multi-tenancy,
+  compatibility with Elasticsearch 8.10 and OpenSearch 2.9/2.10,
```

```suggestion
  compatibility with Elasticsearch 8.10 and OpenSearch 2.9/2.10/2.11,
```
I think we've added `2.11` too 😃